WordPress/WordPressTest/TestUtilities/JSONLoader.swift

```diff
@@ -1,51 +1,46 @@
 
 import Foundation
 
-@objc open class JSONLoader: NSObject {
-    public typealias JSONDictionary = Dictionary<String, AnyObject>
-
-    /**
-    *  @brief      Loads the specified json file name and returns a dictionary representing it.
-    *
-    *  @param      path    The path of the json file to load.
-    *
-    *  @returns    A dictionary representing the contents of the json file.
-    */
-    open func loadFile(_ name: String, type: String) -> JSONDictionary? {
-
-        let path = Bundle(for: Swift.type(of: self)).path(forResource: name, ofType: type)
-
-        if let unwrappedPath = path {
-            return loadFile(unwrappedPath)
-        } else {
-            return nil
-        }
+typealias JSONObject = Dictionary<String, AnyObject>
+
+extension JSONObject {
```

Comment:
One last nitpick: Since the file now holds only `JSONObject`, how would you feel about renaming to `JSONObject.swift` for consistency?

Reply:
Good catch! I missed that. It's renamed in latest commit.